src/core/Akka.Streams/Implementation/IO/FileSubscriber.cs

```diff
@@ -133,8 +158,14 @@ protected override bool Receive(object message)
                         _completionPromise.TrySetResult(IOResult.Failed(_bytesWritten, ex));
                     } 
                     Context.Stop(Self);
-                })
-                .WasHandled;
+                    return true;
+
+                case var msg when _flushCommand != null && ReferenceEquals(_flushCommand, msg):
```

Comment:
Note - need to add an issue to make this programmable from the outside - i.e. a flow stage that can send a `Flush` signal to the `FileSubscriberActor`.